src/context/MotionConfigContext.tsx

```diff
@@ -25,6 +25,20 @@ export interface MotionConfigContext {
      * @public
      */
     transition?: Transition
+
+    /**
+     * Defines the document context to use for the entire tree.
+     *
+     * @public
+     */
+    documentContext: typeof document
```

Comment:
Curious your thoughts on whether this should live in its own context provider or if it's ok in `MotionConfig`? Also, happy to change this if there's a better name or they should be in a single object rather than two separate properties.